charts/argocd-apps/templates/kubescape.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.kubescape }}
+{{- $kubescapesecret := (lookup "v1" "secret" "cluster-services" "govuk-kubescape") }}
```

Comment:
This might lead to a bootstrap issue, but it's not a big deal for now.

The (potential) issue is that there's a race between this chart being deployed and the Secret being populated, and when the Secret doesn't exist, Helm doesn't retry or fail but returns an empty string. So there's a chance the chart may fail to be deployed on cluster turnup and need a manual retry. Who knows whether it'd actually happen in practice though. The solution is not to read secrets in Helm and only refer to them in the kube YAML itself (i.e. `secretKeyRef` or volumes).